### DISCUSSION TITLE
Do react-pdf support bulleted list?

### DISCUSSION THREAD

**Discussion by tuannhtwendeesoft** [diegomura/react-pdf#1679] [Q&A] [answered]
I want to add bulleted list like below to pdf. Do react-pdf support this?
![Snip20220118_1](https://user-images.githubusercontent.com/73807977/149864454-81c8db00-5191-4340-aa4d-4b497ed2914c.png)

**Comment by jeetiss** [comment, marked as answer]
react-pdf doesn't have a `ul` and `li` primitives or equivalent for them, but you can create your components that will look the same

**Reply by jeetiss** [reply]
check the list component in examples
https://github.com/diegomura/react-pdf/blob/master/packages/examples/src/resume/List.js

**Reply by tuannhtwendeesoft** [reply]
You save my day. Thanks a lot!!!!

**Reply by Arctomachine** [reply]
This example is no longer in repository, gives error 404. Is there newer example?

**Reply by barryam3** [reply]
It moved to https://github.com/diegomura/react-pdf/blob/master/packages/examples/vite/src/examples/resume/List.tsx
